packages/dds/sequence/README.md

```diff
@@ -386,6 +386,57 @@ comments.changeProperties(comment.getIntervalId(), { status: "resolved" });
 comments.removeIntervalById(comment.getIntervalId());
 ```
 
+### Interval stickiness
+
+"Stickiness" refers to the behavior of intervals when text is inserted on either side of the interval. A "sticky" interval is one which expands to include text inserted directly adjacent to it.
+
+A "start sticky" interval is one which expands only to include text inserted to the start of it. An "end sticky" interval is the same, but with regard to text inserted adjacent to the end.
+
+For example, let's look at the string "abc". If we have an interval on the character "b", what happens when we insert text on either side of it? In the below diagrams, we represent an interval by putting a caret directly underneath the characters it contains.
```

Comment:
You should add some code translations to this explanation. I'm looking for something like "no stickiness corresponds to `collection.add(1, 1, Stickiness.None)` and places both interval endpoints on 'b'. Left-stickiness corresponds to `collection.add(1, 1, Stickiness.Left)` and places the start endpoint on 'a' and the end endpoint on 'b'"

or something to that effect.

Reply:
The README could use a rewrite in terms of what a `SequencePlace` is and default behavior if numbers are used instead.